en/streaming-integrator/docs/setup/production-checklist.md

```diff
@@ -0,0 +1,44 @@
+# Production Checklist
+
+Once you download and install WSO2 Streaming Integrator, you may need to update the its default configurations based on your requirements.
+
+The changes that you need to make include the following:
+
+## Providing access
+
+Multiple users with various roles in your organization can require access to your WSO2 Streaming Integrator installation to carry out different activities. In order to manage the level of access provided to each user based on their roles, you are required to configure users roles and permissions. For instructions, see [User Management](../admin/user-management.md).
+
+## Securing the Streaming Integrator
+
+WSO2 SI is an open-source product. Therefore, anyone who downloads it has access to the default users and passwords, default keystore settings, etc. Therefore, you are required to update the configurations related to security in order to ensure that your data is secure when you run WSO2 SI in a production environment. For more information, see the following topics:
+
+- WSO2 uses key stores to store cryptographic keys and certificates that are used for various purposes. For more information on how to configure and manage them, see [Working witb Keystores](../admin/working-with-Keystores.md).
```

Comment:
- WSO2 uses key stores to store cryptographic keys and certificates that are used for various purposes. For more information on how to configure and manage them, see [Working with Keystores](../admin/working-with-Keystores.md).